wdio.conf.js

```diff
@@ -326,22 +285,21 @@ exports.config = {
 };
 
 if (process.env.SAUCE) {
-  let sauceCapabilities;
-  if (browser === 'chrome') {
-    sauceCapabilities = Object.assign({}, chromeCapabilities, {
-      name: `react-widget-${suite}`,
-      idleTimeout: 300,
-      commandTimeout: 600,
-      maxDuration: 3600,
-      seleniumVersion: '3.4.0',
-      screenResolution,
-      platform,
-      version
-    });
-  }
-  else {
-    sauceCapabilities = Object.assign({}, firefoxCapabilities, {
-      name: `react-widget-${suite}`,
+  const sauceCapabilities = (remoteName = 'browser') => {
```

Comment:
Changed to a function so `browserLocal` and `browserRemote` could inject their name